docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -63,6 +64,483 @@ We highly recommend using [pm2](https://github.com/Unitech/pm2/) to manage your
+[2019-05-20T14:06:01.556Z] info Version: 3.0.0-alpha.25.2 (node v10.15.3)
+[2019-05-20T14:06:01.556Z] info To shut down your server, press <CTRL> + C at any time
+
+[2019-05-20T14:06:01.557Z] info ☄️  Admin panel: http://localhost:1337/admin
+[2019-05-20T14:06:01.557Z] info ⚡️ Server: http://localhost:1337
+
+```
+
+Minimumly, [create an admin user](http://localhost:8080/documentation/3.x.x/getting-started/quick-start.html#_3-create-an-admin-user).
+
+You will next need to [install and configure PM2 Runtime](#install-and-configure-pm2-runtime).
+
+### Install and configure PM2 Runtime
+
+[PM2 Runtime](https://pm2.io/doc/en/runtime/overview/?utm_source=pm2&utm_medium=website&utm_campaign=rebranding) allows you to keep your Strapi project alive and to reload it without downtime.
+
+Ensure you are logged in as a **non-root** user. You will install **PM2** globally:
+
+```bash
+npm install pm2@latest -g
+```
+
+Navigate to your **Strapi Project folder** and use the following command to set the environment variable to production and start `pm2`:
+
+`Path: ./my-project/`
+
+```bash
+NODE_ENV=production pm2 start --name="strapi" npm -- start
+```
+
+Follow the steps below to have your app launch on system startup. (**NOTE:** These steps are modified from the Digital Ocean [documentation for setting up PM2](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-18-04#step-3-%E2%80%94-installing-pm2).)
+
+- Generate and configure a startup script to launch PM2, it will generate a Startup Script to copy/paste, do so:
+
+```bash
+$ cd ~
+$ pm2 startup systemd
+
+[PM2] Init System found: systemd
+[PM2] To setup the Startup Script, copy/paste the following command:
+sudo env PATH=$PATH:/usr/bin /usr/lib/node_modules/pm2/bin/pm2 startup systemd -u your-name --hp /home/your-name
+```
+
+- Copy/paste the generated command:
+
+```bash
+$ sudo env PATH=$PATH:/usr/bin /usr/lib/node_modules/pm2/bin/pm2 startup systemd -u your-name --hp /home/your-name
+
+[PM2] Init System found: systemd
+Platform systemd
+
+. . .
+
+
+[PM2] [v] Command successfully executed.
++---------------------------------------+
+[PM2] Freeze a process list on reboot via:
+   $ pm2 save
+
+[PM2] Remove init script via:
+   $ pm2 unstartup systemd
+```
+
+- Next, `Save` the new PM2 process list and environment. Then `Start` the service with `systemctl`:
+
+```bash
+pm2 save
+
+[PM2] Saving current process list...
+[PM2] Successfully saved in /home/your-name/.pm2/dump.pm2
+
+```
+
+- **OPTIONAL**: You can test to see if the script above works whenever your system reboots with the `sudo reboot` command. You will need to login again with your **non-root user** and then run `pm2 list` and `systemctl status pm2-your-name` to verify everything is working.
+
+In the sections to follow, are a few recommended additional actions to make developing your project more efficient and to set-up a few additional aspects of your server.
```

Comment:
... and to set-up a few additional **functions on your server.** 

I think is better.  Functions could be **processes**.